### ISSUE TITLE
BMPlayerConfig no longer works

### ISSUE THREAD

**Issue by Jai-Krish** [BrikerMan/BMPlayer#192]
It seems like commit [8822405](https://github.com/BrikerMan/BMPlayer/commit/882240560379355854a3ffb1c00d5c9375b608b8#diff-5614928dc6f88d91362c19abcc91e9ebL290) removed BMPlayerConfig's functionality. Is it intentional?

**Comment by BrikerMan** [comment]
No.. this is a mistake. Will fix later. Thanks for your feedback.
